helm/libero-reviewer/values.yaml

```diff
@@ -0,0 +1,27 @@
+# Default values for libero-reviewer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+images:
+  tag: latest
+  pullPolicy: Always
+  server:
```

Comment:
Is this where we'll add the other services?

Reply:
they will need other `templates/*.yaml` files, but we'll definitely add Docker image names here